backend/infrahub/events/branch_action.py

```diff
@@ -36,20 +40,23 @@ def get_messages(self) -> list[InfrahubMessage]:
         return events
 
 
-class BranchCreateEvent(InfrahubBranchEvent):
+class BranchCreatedEvent(InfrahubEvent):
     """Event generated when a branch has been created"""
 
-    branch_id: str = Field(..., description="The ID of the mutated node")
+    event_name: str = f"{EVENT_NAMESPACE}.branch.created"
+
+    branch_name: str = Field(..., description="The name of the branch")
+    branch_id: str = Field(..., description="The ID of the branch")
     sync_with_git: bool = Field(..., description="Indicates if the branch was extended to Git")
 
     def get_name(self) -> str:
         return f"{self.get_event_namespace()}.branch.created"
```

Comment:
Do we still need the .get_name() method after we added the .event_name attribute? Or should we at least change it to return self.event_name to avoid issues where we forget to update one of the sides?